test/timed_asset_lock_contract_test.py

```diff
@@ -1,60 +1,69 @@
-import sys
-sys.path.append('/home/palmerss/Desktop/SmartContracts/test')
+
+import base64
 import time
 
 import pytest
 from algosdk.v2client import algod
-
 from algosdk import account, mnemonic, constants
 from algosdk.encoding import encode_address, is_valid_address
 from algosdk.error import AlgodHTTPError, TemplateInputError
+from akita_inu_asa_utils import read_local_state, read_global_state, wait_for_txn_confirmation
+
+
+NUM_TEST_ASSET = int(1e6)
+ESCROW_TIME_LENGTH = int(90)
+
 
 @pytest.fixture(scope='class')
 def test_config():
     from test.testing_utils import load_test_config

```

Comment:
In the other cases the import is from testing_utils not test.testing_utils - is there a reason for having both?